src/config.js

```diff
@@ -16,6 +16,8 @@ const config = {
   reporteesMetrics: {
     assigned_defect: {
       label: 'Assigned (defect)',
+      // Higher threshold of assigned defects - to highlight defects on Reportees dashboard
+      assignedDefectsHt: 20,
```

Comment:
what means "Ht" here? can we make it more explicit?

Reply:
Changed key to maxCount